packages/text-input-react/src/TextField.tsx

```diff
@@ -58,9 +69,10 @@ export const TextField = ({
                 readOnly={readOnly}
                 value={value}
                 maxLength={maxLength}
+                style={getLength() ? { width: getLength() } : undefined}
```

Comment:
Maybe prettier if getLength returns the object, then we'll reduce the need of this turnary.

Not a huge fan of inlinestyle, but might not be a great way to get around it here.

Reply:
Sortof the same goes with width killing max-width. In css is the other way around, so we should document that behaviour and again maybe throw a warning/error.

Reply:
I like the idea of returning the style object. I don't particularly like ternaries in props either, just didn't think that far.

I'm not sure exactly what you mean by killing `max-width`? That the inline will override it in specificity? In that case I feel like the intentionality in specifying a width in the prop is enough explanation..

Reply:
Yeah, my comment did not make that much sense. So, just disregard that.